plugins/other/sshd_log

```diff
@@ -28,18 +29,35 @@
 #env.logfile /var/log/messages
 #env.category users
 #
+# config example with journald
+#[sshd_log]
+#group systemd-journal
+#env.logfile journald
+#env.journald_unit sshd.service
+#
 
 LOG=${logfile:-/var/log/secure}
 CATEGORY=${category:-system}
+JOURNALD_UNIT=${journald_unit:-sshd.service}
 
 
 if [ "$1" = "autoconf" ]; then
-        if [ -r "$LOG" ]; then
-                echo yes
-                exit 0
+        if [ $LOG == "journald" ]; then
+                if journalctl -F _SYSTEMD_UNIT | grep $JOURNALD_UNIT -q; then
+                        echo yes
+                        exit 0
+                else
+                        echo no
+                        exit 1
+                fi
         else
-                echo no
-                exit 1
+                if [ -r "$LOG" ]; then
+                        echo yes
+                        exit 0
+                else
+                        echo no
```

Comment:
There is a convention to add some details regarding the failure in brackets, e.g.: "no ($logfile is missing)"